ui/.eslintrc.js

```diff
@@ -38,12 +39,19 @@ module.exports = {
       ],
       parserOptions: {
         sourceType: 'script',
-        ecmaVersion: 2015,
       },
       env: {
         browser: false,
         node: true,
       },
+      plugins: ['node'],
+      rules: Object.assign({}, require('eslint-plugin-node').configs.recommended.rules, {
+        // add your custom rules and overrides for node files here
+
+        // this can be removed once the following is fixed
+        // https://github.com/mysticatea/eslint-plugin-node/issues/77
+        'node/no-unpublished-require': 'off'
```

Comment:
This is because nomad-ui itself is unpublished?

Reply:
Based on the commentary [here](https://github.com/mysticatea/eslint-plugin-node/issues/77), that does seem to be the case. This change is in the [Ember CLI blueprint](https://github.com/ember-cli/ember-cli/commit/626d900215356dc7cc2d7562cf78d42079194f04#diff-d242ba8db51568d33a85c81d0eefe26e) though, so it just came along when I ran `ember-cli-update`.